client/app/scripts/superdesk/notification/notification.js

```diff
@@ -22,6 +34,9 @@
         ws.onmessage = function(event) {
             var msg = angular.fromJson(event.data);
             $rootScope.$broadcast(msg.event, msg.extra);
+            if (_.has(ReloadEvents, msg.event)) {
+                $rootScope.$broadcast('reload', msg);
```

Comment:
I think it would be more explicit if you just inject reload service and call some method there, or are you expecting other consumers?